lib/model.js

```diff
@@ -1,32 +1,32 @@
-var Model = require('hmpo-model');
+'use strict';
 
-var util = require('util'),
-    _ = require('underscore');
+const Model = require('hmpo-model');
+const _ = require('underscore');
 
-var SessionModel = function (attrs, options) {
-    var session = options.session,
-        key = options.key;
+class SessionModel extends Model {
+    constructor(options) {
```

Comment:
I don't think anything inherits from the session model, as it's  not exposed outside of the wizard